docs/sources/reference/run.md

```diff
@@ -790,7 +790,7 @@ command is not available for this logging driver
 
 #### Logging driver: journald
 
-Journald logging driver for Docker. Writes log messages to journald. `docker logs` command is not available for this logging driver
+Journald logging driver for Docker. Writes log messages to journald; the container id will be stored in the journal's `CONTAINER_ID` field. `docker logs` command is not available for this logging driver.
```

Comment:
#### Logging driver: journald

Journald logging driver for Docker. Writes log messages to journald; the
container id is stored in the journal's `CONTAINER_ID` field. The `docker logs`
command is not available for this logging driver. For detailed information on working with this log, see [The journald logging driver](/reference/logging/journald) page in this reference.

-->https://gist.github.com/moxiegirl/4a58ad20a7b0368197b9